src/main/steps/c100-rebuild/child-details/other-children/navigationController.ts

```diff
@@ -0,0 +1,53 @@
+import { Case } from '../../../../app/case/case';
+import { OtherChildrenDetails } from '../../../../app/case/definition';
+import { applyParms } from '../../../common/url-parser';
+import {
+  C100_CHILDERN_OTHER_CHILDREN_NAMES,
+  C100_CHILDERN_OTHER_CHILDREN_PERSONAL_DETAILS,
+  C100_CONFIDENTIALITY_DETAILS_KNOW,
+  PageLink,
+} from '../../../urls';
+
+class OtherChildrenDetailsNavigationController {
+  private otherChildrenDetails: OtherChildrenDetails[] | [] = [];
+
+  private childId: OtherChildrenDetails['id'] = '';
+
+  private getOtherChild(): OtherChildrenDetails | null {
```

Comment:
Please try to reuse the method for both child details & other children as already requested. Try not to write duplicates which does the same thing across features.